demosplan/DemosPlanCoreBundle/Entity/FaqCategory.php

```diff
@@ -13,14 +13,15 @@
 use DateTime;
 use DemosEurope\DemosplanAddon\Contracts\Entities\UuidEntityInterface;
 use demosplan\DemosPlanCoreBundle\Entity\User\Customer;
+use demosplan\DemosPlanCoreBundle\Logic\Faq\FaqCategoryInterface;
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
 use UnexpectedValueException;
 
 /**
  * @ORM\Entity(repositoryClass="demosplan\DemosPlanCoreBundle\Repository\FaqCategoryRepository")
  */
-class FaqCategory extends CoreEntity implements UuidEntityInterface
+class FaqCategory extends CoreEntity implements UuidEntityInterface, FaqCategoryInterface
```

Comment:
I guess "implements UuidEntityInterface," is unnecessary here because FaqCategoryInterface already extends the UuidEntityInterface.
Isn't it?

Reply:
removed in 9a35ce471ecbaa9ad940ebd8954ce70f55999a12